src/server/api/routes/channel-members.router.js

```diff
@@ -5,6 +5,53 @@ const router = express.Router({ mergeParams: true });
 // controllers
 
 const channelMembersController = require('../controllers/channel-members.controller');
+/**
+ * @swagger
+ * /channel-members/common-channels:
+ *  get:
+ *    summary: check which channels have provided users in commone
+ *    description:
+ *      Take an array of users and return the common channels' ids are they member of
+ *    produces: application/json
+ *    parameters:
+ *      - in: query
+ *        name: users
+ *        description: Array of users Ids.
+ *        required: true
+ *        schema:
+ *          type: array
+ *          items:
+ *             type: integer
+ *
+ *    responses:
+ *      201:
+ *        description: Ok
+ *        content:
+ *          application/json:
+ *             schema:
+ *               type : array
+ *               items:
+ *                  type : integers
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/common-channels', (req, res) => {
+  const { users } = req.query;
+  const arrUsers = JSON.parse(users);
+  channelMembersController
+    .checkForCommoneChannels(arrUsers)
```

Comment:
Small typo, it should be `Common` without `e` at the end.
Also, I would name it get instead of check since you are returning something.

Prefer `users` to `arrUsers`.